MapleServer2/Commands/Game/GMCommands.cs

```diff
@@ -26,4 +31,64 @@ public override void Execute(GameCommandTrigger trigger)
             trigger.Session.SendNotice("Oneshot mode enabled.");
         }
     }
+    public class SetJobCommand : InGameCommand
+    {
+        public SetJobCommand()
+        {
+            Aliases = new()
+            {
+                "setJob",
+                "sj"
+            };
+            Description = "Sets character's job, e.g. /setJob assassin 1";
+            AddParameter<string>("job", "Classname, e.g.: striker");
+            AddParameter<byte>("awakened", "Awakened = 1, Unawakened = 0");
+        }
+
+        public override void Execute(GameCommandTrigger trigger)
+        {
+
+            string jobName = trigger.Get<string>("job");
+            byte awakened = trigger.Get<byte>("awakened");
+
+            Player player = trigger.Session.Player;
+            IFieldObject<Player> fieldPlayer = trigger.Session.FieldPlayer;
+
+            long activeSkillTabId = player.ActiveSkillTabId;
+            SkillTab skillTab = player.SkillTabs.First(tab => tab.TabId == activeSkillTabId);
+
+            Job job = Job.None;
+
+
+            if (!Enum.TryParse(jobName, true, out job))
+            {
+                player.Session.SendNotice($"{jobName} is not a valid class name");
+                return;
+            }
+
+            if (Job.None == job)
```

Comment:
This if will ever get reached since the if above returns if it can't parse? Only if you use `/sj None` correct?